resources/default_pipeline_environment.yml

```diff
@@ -395,6 +395,11 @@ steps:
         && mkdir -p \$GOPATH/src/${config.whitesource.projectName.substring(0, config.whitesource.projectName.lastIndexOf('/'))}
         && ln -s \$(pwd) \$GOPATH/src/${config.whitesource.projectName}
         && cd \$GOPATH/src/${config.whitesource.projectName} && dep ensure
+    dub:
+      buildDescriptorFile: './dub.json'
+      stashContent:
```

Comment:
@andre2007 Famous last demands: Would there be a publicly available docker image which we could use for the dub scenario and what would be the related workspace? Any additional installs needed? I think you have a working scenario but not necessarily on an official image available to anyone.

Reply:
@nevskrem the docker images only serves a purpose if there is a functionality in WhiteSource unified agent which calls e.g. the respective package manager (pip/npm/...).
In case of D, there is no specific code in unified but the generic logic (files names / hashes) is used. Therefore starting a docker image would serve no purpose. Should I anyway add the official docker image?